### DISCUSSION TITLE
Is conversation context included in the query?

### DISCUSSION THREAD

**Discussion by seoeaa** [safevideo/autollm#97] [Q&A] [answered]
The conversation context is not included in the search does not work

**Comment by talhasarit** [comment]
Hi @seoeaa, can you please elaborate on the error you are encountering with the corresponding code snippets or screenshots? It would help us a lot to assist you better.

**Comment by fcakyon** [comment, marked as answer]
Hello @seoeaa, we appreciate your feedbacks on `autollm` 🙏 

Conversation context is included in the search, you can see the default prompt template here: https://github.com/safevideo/autollm/blob/69ff4be60f215debaaefe85e4b7c0e4b836301be/autollm/utils/templates.py#L10-L19
